### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by leonuetzel** [flandreas/antares#315] [General]
Hello,

where can i donate some Money?
I love this Project and i want to support the great work and effort flandres put into it.

**Comment by flandreas** [comment]
@leonuetzel This is very kind of you, thanks a lot! There is currently no way for donating money. You are already supporting the project a lot by reporting bugs and feature requests, for which I'm very thankful. If you would like to do even more, recommend Antares to your friends or spread the word in any other way that could help Antares to get known better.
